packages/components/pie-modal/src/modal.scss

```diff
@@ -137,18 +151,31 @@
         --modal-button-spacing: var(--dt-spacing-d);
         --modal-footer-padding: var(--dt-spacing-d);
 
+        grid-area: footer;
+        min-block-size: var(--dt-spacing-e);
         display: flex;
         flex-flow: row-reverse;
         flex-wrap: wrap;
         gap: var(--modal-button-spacing);
+    }
+
+    &[hasactions] .c-modal-footer {
         padding: var(--modal-footer-padding);
 
         @media (min-width: $breakpoint-wide) {
             --modal-footer-padding: var(--dt-spacing-e);
         }
     }
 
+
+    & .c-modal-scrollContainer,
+    &[isfooterpinned] .c-modal-content {
```

Comment:
Are we not creating a double scrollbar with this?

Does this not result in:

<div class=".c-modal-scrollContainer"> -> Scrollable
 <div isFooterPinned>
  <div class="c-modal-content"></div> -> Scrollable
  </div>
</div>

Reply:
If `isFooterPinned` is false, the markup is:

```html
<dialog class="c-modal">
    <header class="c-modal-header">
    <!--  -->
    </header>
    <div class="c-modal-scrollContainer">  <-- this is the only element with a scroll bar
        <article class="c-modal-content c-modal-content--scrollable">
        <!--  -->
        </article>
        <footer class="c-modal-footer">
        <!--  -->
        </footer>
    </div>
</dialog>
```

When `isFooterPinned` is true,
```html
<dialog class="c-modal">
    <header class="c-modal-header">
    <!--  -->
    </header>
    <article class="c-modal-content c-modal-content--scrollable">   <-- this is the only element with a scroll bar
    <!--  -->
    </article>
    <footer class="c-modal-footer">
    <!--  -->
    </footer>
</dialog>
```